quantus/helpers/enums.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+
+
+class DataType(Enum):
+    """
+    This enum represents the different types of data that a metric implementation currently supports.
+
+        - IMAGE: Represents image data.
+        - TABULAR: Represents tabular data.
+        - TEXT: Represents text data.
+    """
+
+    IMAGE = "image"  # 3D data
```

Comment:
`IMAGE`, `TIME-SERIES` and `TABULAR` describe the shapes of the tensors.
While `TEXT` the underlying data type.

Those two look like separate, independent categories to me.
Ideally, we should also specify which data types we expect (or support) `float32`, `float64` or something else?

And finally, for images it would be really useful to make it visible if we expect 
pixel values in the range [0, 255] or [0, 1]. This one always leaves me puzzled 🙃

Reply:
> IMAGE, TIME-SERIES and TABULAR describe the shapes of the tensors. While TEXT the underlying data type.

I agree, the text one is vague. I kept it there more as a placeholder until we further define it with an actual usecase.

> Those two look like separate, independent categories to me.

Could be so, should we develop it further when we have the first text model wrapped into Quantus?

> Ideally, we should also specify which data types we expect (or support) float32, float64 or something else?

We should definitely get more precise. Should we define this in a separate PR? I would say that this PR does a first good basic job and then we can go more in detail in a follow-up PR.

> And finally, for images it would be really useful to make it visible if we expect pixel values in the range [0, 255] or [0, 1]. This one always leaves me puzzled 🙃

Hmmm. I think this is out of the scope of Quantus since it is related to the model training process..... e..g, whether the model is trained on [0,1] and not [0, 255] then, ultimately this is also what should be passed as input on test (explanation/ evaluation) time.

Reply:
Sounds good, I will do a proposal PR with shapes and `dtypes` differentiation on top of this one in the next time then